tutorials/export/android_gradle_build.rst

```diff
@@ -57,3 +57,20 @@ configuration is needed.
 
     For example, ``_example/image.png`` will **not** be included as an asset,
     but ``_image.png`` will.
+
+Disable Swipe-to-Dismiss on Wear OS
+-----------------------------------
+
+To disable swipe-to-dismiss functionality on Wear OS devices, follow these steps:
+
+1. Open the file ``res://android/build/res/values/themes.xml`` in your file editor.
+2. Add or update the ``android:windowSwipeToDismiss`` attribute to ``false`` in the `GodotAppMainTheme` style.
```

Comment:
```suggestion
2. Add or update the ``android:windowSwipeToDismiss`` attribute to ``false`` in the ``GodotAppMainTheme`` style.
```
Like this if you want `code`

```suggestion
2. Add or update the ``android:windowSwipeToDismiss`` attribute to ``false`` in the *GodotAppMainTheme* style.
```
Like this if you actually want *italics*

In RST single backticks does "default style" which is a link/reference if you also put _ at the end, and italics otherwise

Also, it's currently slightly ungrammatical, since "Add the attribute to false" isn't grammatical. But the grammatically correct options both seem a bit more awkward, so IDK, it's maybe fine as-is:
```suggestion
2. Add the ``android:windowSwipeToDismiss`` attribute in ``GodotAppMainTheme`` style if you don't already have it. Then set it to ``false``.
```
```suggestion
2. Update the ``android:windowSwipeToDismiss`` attribute to ``false`` in the `GodotAppMainTheme` style. If you don't already have it, add it first.
```